src/components/new-request/search.vue

```diff
@@ -133,26 +145,6 @@
       </template>
     </v-row>
 
-    <!-- Xpro/Federal bullets -->
-    <!-- *** TODO: move this into flows above -->
-    <!-- *** TODO: simplify logic -->
-    <!-- <template v-if="(getEntityTypeCd || isFederal || isRestorable) && (isNamedCompany || isAmalgamation)">
-      <v-row v-if="isXproFlow && isFederal">
-        <XproFederalBullets />
-      </v-row>
-    </template> -->
-
-    <!-- Corporate Number checkbox, only for Canadian MRAS jurisdictions -->
-    <!-- *** TODO: move this into flows above -->
-    <!-- *** TODO: simplify logic -->
-    <!-- <template v-if="(getEntityTypeCd || isFederal || isRestorable) && (isNamedCompany || isAmalgamation)">
-      <v-row v-if="isCanadian && isMrasJurisdiction">
-        <v-col cols="12" class="d-flex justify-end">
-          <CorpNumberCheckbox />
-        </v-col>
-      </v-row>
-    </template> -->
-
```

Comment:
I'm assuming you have taken the code you want from here so I'm now deleting this.